test/exec.test.js

```diff
@@ -88,6 +91,25 @@ require('ut-run').run({
                 error(error, assert) {
                     assert.equal(error.type, 'bus.methodNotFound', 'SP is private because it starts with _');
                 }
+            },
+            {
+                method: 'test.test.fetch',
+                params: {
+                    'data.content': Date.now().toString()
+                },
+                result(result, assert) {
+                    assert.ok(Array.isArray(result.data) && !result.data.length, 'no records found');
+                }
+            },
+            {
+                method: 'test.test.fetch',
+                params: {
+                    'data.content': 'abc'
+                },
+                result(result, assert) {
+                    assert.ok(Array.isArray(result.data) && result.data.length === 1, 'record found');
+                    assert.equal(result.data[0].txt, 'abcde', 'the found record matches expectation');
```

Comment:
A bit more tests will not hurt. Maybe search for 'cde' and see if two rows are returned. You may need to add `ORDER BY` in the procedure to get consistent results.

Reply:
[done](https://github.com/softwaregroup-bg/ut-port-sql/commit/33aa5f95966ab566e8cae9f659b904429c8c2831)